service.policy.json

```diff
@@ -0,0 +1,6 @@
+{
+    "properties": [],
+    "constraints": [
+        "preferredLanguage == Python AND purpose == learning"
```

Comment:
The service policy will almost certainly be created by the service developer. This person would definitely know the language and perhaps system requirements of the nodes that are trying to run the service. So I think we should **only** have the `"preferredLanguage == Python"` constraint in here. With these constraints split up into the service and deployment policy they are combined and evaluated during the agreement process and will only deploy to nodes that have both in their node properties.

Reply:
hi @t-fine i made the changes. You also mentioned something about rebasing?

Reply:
@adhishreekadam awesome, looking good. So yes you will need to rebase this PR on top of the changes that have been merged to the Makefile and README file ... I use this bash function:
```troyfine@Troys-MacBook-Pro-2 devops-1 % whence -f urebase 
urebase () {
	if [ -z $1 ]
	then
		echo "Pass in upstream branch to rebase off of"
		return 1
	else
		echo "Rebase ontop of upstream $1?[y/N]:"
		read RESPONSE
		if [[ $RESPONSE == y ]]
		then
			git pull --rebase upstream $1
		fi
	fi
}

When you're on your `Issue-26` branch you can execute it with `urebase main` and it will attempt to merge the the changes in with yours ... chances are you will have some merge conflicts though, so you will have to manually fix those. It will show you the files where there are conflicts and you will see incoming and current changes between `>>>>>` and `<<<<<` so you just have to leave only the code changes we want and remove the `<` and `>` characters (and whatever text) then run `git rebase --continue` (might be slightly different) until it says `Successfully rebased` and then you follow it up with a `git push -f` and you're done!

Reply:
I have fixed the merge conflicts.

Reply:
Thanks! I have fixed the merge conflicts.